src/sagemaker/processing.py

```diff
@@ -269,7 +269,9 @@ def _normalize_outputs(self, outputs=None):
 
 
 class ScriptProcessor(Processor):
-    """Handles Amazon SageMaker processing tasks for jobs using a machine learning framework."""
+    """Handles Amazon SageMaker processing tasks for jobs using a machine learning framework,
+    which allows for providing a script to be run as part of the Processing Job.
```

Comment:
nit [pep257](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings): one line first line, details can be added in body